tests/e2e/test_io_performance.py

```diff
@@ -35,22 +35,47 @@ def base_setup(self, request, interface_iterate, pod_factory):
 
     @pytest.mark.parametrize(
         argnames=[
-            "size", "io_direction", "jobs",
+            "size", "io_direction", "bs", "jobs",
             "runtime", "depth", "sheet_index"
         ],
         argvalues=[
             pytest.param(
-                *['1GB', 'rw', 1, 60, 4, 1],
+                *['1000GB', 'rw', '4k', 1, 1200, 4, 4],
                 marks=pytest.mark.polarion_id("OCS-676")
             ),
             pytest.param(
-                *['1GB', 'rw', 6, 60, 16, 2],
+                *['1000GB', 'rw', '4k', 4, 600, 8, 5],
```

Comment:
since we are changing the numjobs and io depth, we need to change the polarion test case as well. may be we can ask respective owner once PR is merged.

Reply:
@vavuthu agree, just testing this out in wip branch first locally.

Reply:
@vasukulkarni , Correct me if I am wrong, I am not sure about the complete workflow, but are the PVCs configured to handle 1000GB from ocs-ci ? I mean are we creating the storageclass which handles this size?

Reply:
I haven't pushed the pvc change at, here its 3G, Just running it locally and had filed a bz because it was overprovisioned(thin provisioning). From the pod side we will not see any complain, bz is 1764302.

Reply:
@vasukulkarni , i think that you need also to change the pvc size.
i did it in my local env. i can share with you my code.

Reply:
@avilir I changed here ocs_ci/templates/pv_pvc/PersistentVolumeClaim.yaml , Do you see any other place it needs change?

Reply:
@vasukulkarni I think that you shouldn't change the template, the better is to change the test by creating the PVC with new size.

    def base_setup(self, request, interface_iterate, pvc_factory, pod_factory):
        self.interface = interface_iterate
        self.pvc_obj = pvc_factory(size=300)
        self.pod_obj = pod_factory(interface=self.interface, pvc=self.pvc_obj)